plaidbench.py

```diff
@@ -80,6 +78,111 @@ def has_plaid():
         return False
 
 
+def value_check(examples, epochs, batch_size):
+    if epochs > examples:
+        raise ValueError('The number of epochs must be less than the number of examples.')
+    if batch_size > (examples // epochs):
+        raise ValueError('The number of examples per epoch must be greater than the batch size.')
+    if examples % epochs != 0:
+        raise ValueError('The number of examples must be divisible by the number of epochs.')
+    if (examples // epochs) % batch_size != 0:
+        raise ValueError('The number of examples per epoch is not divisble by the batch size.')
+
+
+def train(x_train, y_train, epoch_size, model, batch_size, compile_stop_watch, 
+    epochs, stop_watch, output, network):
```

Comment:
Either align these vertically (so put 'epochs' under 'x_train'), or use a hanging indent with four spaces and _no_ parameters on the first line.  Here's the [style guide rule](https://google.github.io/styleguide/pyguide.html?showone=Indentation#Indentation).